src/ai/default/ca.cpp

```diff
@@ -277,9 +277,9 @@ move_leader_to_goals_phase::~move_leader_to_goals_phase()
 double move_leader_to_goals_phase::evaluate()
 {
 
-	const config &goal = get_leader_goal();
+	config goal = get_leader_goal();
 	//passive leader can reach a goal
-	if (!goal) {
+	if ((false)) {
```

Comment:
Any reason not to use an `optional_child` in `get_leader_goal()`? I guess that's worked into the aspects system so it might be easier to test here `if(goal.empty())`.